package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "linter-pylint",
   "main": "./lib/main.js",
-  "version": "2.1.1",
```

Comment:
Please don't change the version in a PR, that will be done when releasing an update.